src/reducers/books.js

```diff
@@ -0,0 +1,37 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/index';
+
+const initialState = [
+  {
+    id: 100 * Math.random(),
```

Comment:
**[OPTIONAL]**
The use of the expression `100 * Math.random()` does not guarantee unique number generation, you can use existing libraries like `uuid`, `short-id`